src/libstd/net/tcp.rs

```diff
@@ -630,6 +630,8 @@ impl TcpListener {
     /// listener.set_only_v6(true).expect("Cannot set to IPv6");
     /// ```
```

Comment:
Similarly, there are a couple other references to `set_only_v6` in this file that should probably be removed.